### ISSUE TITLE
SA1135 false positive in 1.1.0 Beta 7 for using aliases

### ISSUE THREAD

**Issue by tdworshak** [DotNetAnalyzers/StyleCopAnalyzers#2690]
The following line generates a SA1135 false positive, despite being fully qualified:

```
using ExampleOfFalsePositive = System.ValueTuple<System.Collections.List, int>;
```

**Comment by tdworshak** [comment]
The C# compiler currently requires (and enforces) using aliases to be fully qualified, so StyleCop can ignore using aliases completely.

**Comment by sharwell** [comment]
Names only need to be enforced when they are placed outside. Things like this work:

```
namespace System {
  using ExampleOfFalsePositive = ValueTuple<Collections.IList, int>;
}
```

**Comment by sharwell** [comment]
@tdworshak I have not been able to reproduce this issue.

**Comment by sharwell** [comment]
I added two tests for this issue in #2697, but no changes were made as a result since the tests already pass. Please review the tests and update the issue if there is another way to reproduce a false positive.

**Comment by vweijsters** [comment]
I can reproduce the issue with

``` C#
namespace Test
{
    using ExampleOfFalsePositive = System.ValueTuple<System.Collections.IList, int>;
}
```

The following warning is produced:
```warning SA1135: Using directive for type '(System.Collections.IList, int)' should be qualified```

**Comment by tdworshak** [comment]
Thanks guys.  I appreciate you taking the time to look at this issue!

**Comment by vweijsters** [comment]
Reopening this, as it got closed while merging #2697, but the issue is not resolved.

**Comment by sharwell** [comment]
@vweijsters Awesome, I missed that a new repro case was added.

**Comment by vweijsters** [comment]
@sharwell I've been doing some analysis on this and it only fails when using Microsoft.CodeAnalysis 2.6.0 code, not with the 1.2.1 code.

The [code](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1135UsingDirectivesMustBeQualified.cs#L85):
```C#
var symbol = context.SemanticModel.GetSymbolInfo(usingDirective.Name, context.CancellationToken).Symbol;
```

returns a symbol with a proper name for v1.2.1. It returns a symbol without a name for v2.6.0. It looks to me like a bug in Roslyn, but what are your thoughts about it?
